icubam/www/server.py

```diff
@@ -40,17 +39,16 @@ def make_routes(self):
     kwargs = dict(config=self.config, db_factory=self.db_factory)
     self.add_handler(home.HomeHandler, **kwargs)
     self.add_handler(home.MapByAPIHandler, **kwargs)
-    self.add_handler(db.DBHandler, **kwargs)
-    self.add_handler(VersionHandler, **kwargs)
-    self.add_handler(consent.ConsentHandler, **kwargs)
     self.add_handler(
-      upload_csv.UploadHandler, **{
+      db.DBHandler, **{
```

Comment:
I don't know what the `db` endpoint was for before, didn't it make sense to keep it separated from the upload endpoint?

Reply:
It's our general API endpoint, so I think it's good to keep these together.